common/navbar.js

```diff
@@ -83,7 +94,7 @@
           menuTarget.style.top = parseInt((immediateParent.offsetTop + parent.offsetTop) - parent.scrollTop) + 10 + 'px';
       }
 
-      menuTarget.style.left = parseInt(posValues + immediateParent.offsetWidth) + 5 + 'px';
+      menuTarget.style.left = parseInt(posValues + immediateParent.offsetWidth) + 'px';
```

Comment:
Removed the additional 5px as the cascading dropdowns showed an unnecessary gap in between currently
 
![image](https://user-images.githubusercontent.com/78573171/120389038-c2b47000-c2e0-11eb-9932-5bbd37e1cede.png)